lib/velum/secrets.rb

```diff
@@ -0,0 +1,20 @@
+require 'securerandom'
+
+module Velum
+  # If there is a rails key base stored on disk, return the stored one.
+  # Otherwise, create a new key base from cryptographic random pool and store it.
+  def self.create_secret_key_base(pathname)
+    key_base = nil
+    if pathname.exist?
+      content = IO.read(pathname)
```

Comment:
I know for sure that you've already used it ;-) e.g. `Rails.root` is a `Pathname` object